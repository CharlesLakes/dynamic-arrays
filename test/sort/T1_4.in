10
70 32 48 24 100 33 16 1 28 58