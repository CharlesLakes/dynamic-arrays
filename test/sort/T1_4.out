1 16 24 28 32 33 48 58 70 100